server/game/cards/03_TWI/units/LethalCrackdown.ts

```diff
@@ -0,0 +1,39 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { CardType, RelativePlayer, WildcardRelativePlayer } from '../../../core/Constants';
+
+export default class LethalCrackdown extends EventCard {
+    protected override getImplementationId () {
+        return {
+            id: '1389085256',
+            internalName: 'lethal-crackdown'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'Defeat a non-leader unit. Deal damage to your base equal to that unit\'s power.',
+            targetResolver: {
+                cardTypeFilter: CardType.BasicUnit,
```

Comment:
I think we want to change this to the wildcard non leader unit, because it should be able to defeat tokens as well